cfme/intelligence/reports/schedules.py

```diff
@@ -168,6 +168,16 @@ def queue(self):
         view = navigate_to(self, "Details")
         view.configuration.item_select("Queue up this Schedule to run now")
 
+    @property
+    def status(self):
+        view = navigate_to(self.parent, "All")
+        for item in view.schedules_table.read():
+            if item['Name'] == self.name:
+                if item['Active'] == 'True':
+                    return True
+                else:
+                    return False
```

Comment:
I think best to go with property name `is_enable()` rather than `status`. it will give meaning to this property for checking `schedule` is `enable` or `disable`. 

`schedule.is_enable` will return `True` if its enable else `False`.

Reply:
I like that idea.   I looked at this earlier and I could understand it and it was pretty straightforward, however your suggestion would reduce the line noise and make it even more readable.

Reply:
Great call guys, but I'd prefer `enabled` in this context. Like our use of `self.exists` elsewhere in the framework.

Reply:
There's no need to split the return out here, you can return the result of the statement that your `if` is working off of.

```python
for item in view.schedules_table.read():
    if item['Name'] == self.name 
        return item['Active'] == 'True'
```